ui/components/Core/CorePage.tsx

```diff
@@ -25,7 +36,7 @@ export default function CorePage(props: Props): ReactElement {
             align-items: center;
             background-color: var(--hunter-green);
             z-index: 10;
-            height: ${hasTopBar ? "480px" : "100vh"};
+            height: calc(100vh - ${barSpace}px);
```

Comment:
I would do it inline but it's up to you 
```suggestion
            height: calc(
              100vh - ${(hasTabBar ? 56 : 0) + (hasTopBar ? 64 : 0)}px
            );
```